### ISSUE TITLE
If a reviewer has previously unassigned the review, don't reassign it to them

### ISSUE THREAD

**Issue by gsnedders** [web-platform-tests/wpt-pr-bot#54]
#51 started automatically assigning issues to reviewers.

Some directories are a bit large, and some reviewers might not feel comfortable reviewing everything.

As a first rough approximation, if the issue has previously been assigned to [X] we should assign it to [X] to review.

Note this isn't perfect, because we might also want to unassign PRs if someone is travelling, for example.

**Comment by foolip** [comment]
Good idea. This can be determined from the PR itself and doesn't require storing state in wpt-pr-bot.

Can you file a separate issue for vacation logic if you have a proposed fix?

**Comment by gsnedders** [comment]
> Can you file a separate issue for vacation logic if you have a proposed fix?

I don't, sadly. That I believe is a much harder problem, esp. given it seems unlikely that orgs will be willing to expose any vacation data they have for their employees (and that doesn't work for people not in those orgs).

**Comment by zcorpan** [comment]
More generally, I think the bot should only assign once per PR. If I unassign the person the bot had assigned, it's annoying for the bot to automatically assign somebody else. (e.g. https://github.com/web-platform-tests/wpt/pull/14703 )
